### ISSUE TITLE
I can't download the PDF again on Safari

### ISSUE THREAD

**Issue by bilalmotiwala** [eKoopmans/html2pdf.js#386]
Hi there, I am using HTML2PDF.JS to make a PDF from an HTML page that I have but I am facing some issues on it.

My code looks like this and it works perfectly on chrome/firefox/etc except Safari. Could you please help?

In the commented part I was trying to just save it without opening, it works perfectly everywhere except on Safari where it works for the first time but then it doesn't work at all after that until a session is restarted.

```
function ExportToPDF(element){
                var elementA = $(element)[0];
                html2pdf().from(elementA).toPdf().get('pdf').then(function (pdf) {
                    window.open(pdf.output('bloburl'), '_self');
                    /*pdf.save('PaySlip.pdf');*/
                    /*window.location.reload();*/
                });
            }
```
I'm experimenting around with things that I'm somehow probably not even understanding but I'd really appreciate the help on anyone who is aware of how this issue can be solved. Safari is the only browser causing this issue on both iOS and MacOS.

**Comment by klashman** [comment]
I had the same problem, do you fix it?

**Comment by rajan-g** [comment]
Yes, I have the same issue, Please help me to resolve this issue
